src/cgt/include/cgt/RandomBits.h

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025, The OpenROAD Authors
+
+#pragma once
+
+#include <cstdint>
+#include <random>
+
+namespace cgt {
+
+class RandomBits
+{
+ private:
```

Comment:
The private section should come after the public https://google.github.io/styleguide/cppguide.html#Declaration_Order